plugins/push/.gitignore

```diff
@@ -1 +1,2 @@
-api/parts/apn/build
\ No newline at end of file
+api/parts/apn/build
+frontend/public/stylesheets/main.css
```

Comment:
i believe main.css should not be added to gitignore, since its the legacy stylesheet that does not come from a scss.
So we would need it in the repo.

Reply:
I have renamed the non scss file to old.main.css and instead created a main scss plugin file. As a consequence, I moved it to gitignore.

Reply:
what is the purpose of creating main.scss?

Reply:
also if you renamed it to old.main.css shouldn't it be also included pr. i couldn't find it here.
Besides that we would also have to add it to dashboards.html and Gruntfile afaik.
am i missing something?

Reply:
1. does not each plugin have a main styles file or am I missing something?
2. it's in the pr but since there were not changes, it's been renamed only. I left it until I finish off all the view components of the push plugin, then I will remove it (same with other old view components).

Reply:
okay sorry. got it.
i thought the .gitignore you used was global one.

Reply:
no problem. I am happy that we cleared out the confusion.